code/modules/mob/living/simple_animal/friendly/drone/_drone.dm

```diff
@@ -121,6 +123,24 @@
 	"<span class='notice'>     - Interacting with non-living beings (dragging bodies, looting bodies, etc.)</span>\n"+\
 	"<span class='warning'>These rules are at admin discretion and will be heavily enforced.</span>\n"+\
 	"<span class='warning'><u>If you do not have the regular drone laws, follow your laws to the best of your ability.</u></span>"
+	/// blacklisted drone areas, direct
+	var/list/drone_area_blacklist_flat = list(/area/engineering/atmos, /area/engineering/atmospherics_engine, /area/engineering/supermatter)
+	/// blacklisted drone areas, recursive/includes descendants
+	var/list/drone_area_blacklist_recursive
+	/// whitelisted drone machines, direct
+	var/list/drone_machinery_whitelist_flat
+	/// whitelisted drone machines, recursive/includes descendants
+	var/list/drone_machinery_whitelist_recursive = list(	/obj/machinery/atmospherics, /obj/machinery/autolathe, /obj/machinery/cell_charger, /obj/machinery/disposal, \
+															/obj/machinery/drone_dispenser, /obj/machinery/light, /obj/machinery/pipedispenser, /obj/machinery/recharger, \
+															/obj/machinery/rnd/production)
+	/// whitelisted drone items, direct
+	var/list/drone_item_whitelist_flat = list(	/obj/item/crowbar/drone, /obj/item/screwdriver/drone, /obj/item/wrench/drone, \
+												/obj/item/weldingtool/drone, /obj/item/wirecutters/drone )
```

Comment:
why are multitools not in this list?

Reply:
I purposely left them out for now because I don't see them being necessary, and want to keep drones from being OP at first. I am open to adding them as well if they are acceptable, but probably not as a built in tool, just allow drones to use a normal multitool.

Reply:
I can see multitools necessary to fix telecomms, but maybe drones shouldn't be fixing telecomms.

Reply:
Its best to give them the multitool mostly just because derelict drones can and should build an ore silo

Reply:
Derilict drones can be modified separately

Reply:
wait, are you making derelict drones shy as well, or will they remain as they were?

if the former, wow, this absolutely guts them.

Reply:
I haven't touched derelict drones with this PR, that's a good idea to test out